lib/isafw/isafw.py

```diff
@@ -30,7 +30,7 @@
 
 import sys
 import traceback
-import isafw.isaplugins as isaplugins
+import isaplugins
```

Comment:
Because of the "absolute_import", "import isaplugins" failed in https://ostroproject.org/jenkins/job/dsl-meta-security-isafw_pull-requests/77/